test/openlr.cc

```diff
@@ -71,12 +73,12 @@ TEST(OpenLR, Decode) {
 
     EXPECT_NEAR(locRef.getFirstCoordinate().lng(), fixture.expectedFirstCoordinateLongitude, 1e-5);
     EXPECT_NEAR(locRef.getFirstCoordinate().lat(), fixture.expectedFirstCoordinateLatitude, 1e-5);
-    EXPECT_NEAR(locRef.first.bearing, fixture.expectedFirstCoordinateBearing, 1e-5);
```

Comment:
all of these changes to `first` and `last` is just a refactor as mentioned below